root.yml

```diff
@@ -388,7 +388,7 @@ Resources:
     Condition: CreateSingletonStacks
     Properties:
       NotificationARNs:
-        - !GetAtt NotificationsStack.Outputs.OpsDebugMessagesSNSTopicArn
+        - !GetAtt NotificationsStack.Outputs.OpsWarnMessagesSNSTopicArn
```

Comment:
This we actually want to keep as *debug*, since only CloudFormation-related messages get sent to the Notification ARN. The topic that gets used for our other explicit resources, like CW alarms, needs to get passed in as a param